index.d.ts

```diff
@@ -0,0 +1,33 @@
+// TypeScript Version: 3.6.3
+
+interface Options {
+  onlyFirst: boolean;
```

Comment:
This option needs a doc comment. Seehttps://github.com/sindresorhus/typescript-definition-style-guide#documentation